source/reference/minio-mc-admin/mc-admin-heal.rst

```diff
@@ -15,15 +15,13 @@ Description
 
 .. start-mc-admin-heal-desc
 
-The :mc-cmd:`mc admin heal` command scans for objects that are damaged or
-corrupted and heals those objects.  
+The :mc-cmd:`mc admin heal` command scans for objects that are damaged or corrupted and heals those objects.  
 
 .. end-mc-admin-heal-desc
 
-:mc-cmd:`mc admin heal` is resource intensive and typically not required even
-after drive failures or corruption events. Instead, MinIO automatically heals
-objects damaged by silent bit rot corruption, drive failure, or other issues on
-``POST`` or ``GET``. MinIO also performs periodic background object healing using the :ref:`scanner <minio-concepts-scanner>`.
+:mc-cmd:`mc admin heal` is resource intensive and typically not required even after drive failures or corruption events. 
+Instead, MinIO automatically heals objects damaged by silent bit rot corruption, drive failure, or other issues on ``POST`` or ``GET``. 
+MinIO also performs periodic background object healing using the :ref:`scanner <minio-concepts-scanner>`.
```

Comment:
We actually automatically begin healing aggressively on drive replacement. So

- mc admin heal is resource intensive and typically not required as a manual process
- MinIO instead automatically heals damaged objects...
- It does periodic object healing, and...
- It aggressively heals after drive replacement

Reply:
Converted to a list with the above suggestions.